### PR TITLE
AutomateWoo ChartMogul integration base logic.

### PR DESCRIPTION
**Pull request by saas786**
Test:

### 1) Setup workflows to test triggers, such as

a) Customer Created.
b) Order Created.
c) Order status changed.
d) Subscription created.
e) Subscription status changed.
f) Subscription cancelled.

### CODE REVIEW

File: src/AutomateWoo/Actions/CustomerCreated.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			return;
+
+		} catch ( \Throwable $e ) {
+			Logger::log()->error( $e->getMessage() );
+		}
+
+		$this->workflow->log_action_error( $this, 'Exporting new customer failed. Please verify action config.' );
+	}
+
+	/** @since 2021.06.29 */
```
Comment (paulkirspuu) [inline src/AutomateWoo/Actions/CustomerCreated.php:91]: comment alignment is off, for all similar files as well.

File: src/AutomateWoo/Actions/CustomerCreated.php [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	/**
+	 * Loads action's fields.
+	 *
+	 * @since 2021.06.29
+	 */
+	public function load_fields(): void { // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+		$this->add_field( $this->get_field_data_source() );
+	}
+
+	/**
```
Comment (paulkirspuu) [inline src/AutomateWoo/Actions/CustomerCreated.php:43]: description of method missing, same thing for all similar files.

**Review by paulkirspuu — CHANGES_REQUESTED** [review]
added a couple of comments.

overall looks good, no critical errors at first sight. if everything is tested, CXL coding standards applied, implementation QA'd by a second pair of eyes, and end result is as intended, id say its good to go.

File: src/AutomateWoo/Actions/CustomerCreated.php (hunk 2 above)
Comment (saas786) [reply src/AutomateWoo/Actions/CustomerCreated.php:44]: @paulkirspuu
Take a look at both changes now...

Comment (paulkirspuu) [reply src/AutomateWoo/Actions/CustomerCreated.php:44]: alright, all good on my end now, thanks!

**Pull request merged**